docs/providers/aws/examples/hello-world/fsharp/serverless.yml

```diff
@@ -0,0 +1,84 @@
+# Welcome to Serverless!
```

Comment:
Can you please add a `README.md` file for this example like it's done with the C# example:

https://github.com/serverless/serverless/blob/master/docs/providers/aws/examples/hello-world/csharp/README.md